### PR TITLE
Ascii to hex

### PR DESCRIPTION
**Pull request by vishalbhowmick**
I added ascii-hex converter and it works both ways.
Screenshot 2023-10-06 000250

### CODE REVIEW

**Comment by vaibhav-kaushal** [comment]
@vishalbhowmick Can it handle UTF-8 characters too?

**Comment by vishalbhowmick** [comment]
@HariKrishna-28 can you give it a review

File: src/components/Sidebar/sidebarLists.js [hunk 1]
```diff
@@ -60,6 +60,12 @@ export const sideBarList = [
     route: "/json-converter",
     tags: "Conversion,Syntax,JSON",
   },
+  {
+    name: "ASCII-Hex Converter",
+    iconClass: "bi bi-hexagon",
+    route: "/ascii-to-hex",
+    tags: "Conversion,Syntax,JSON",
```
Comment (vaibhav-kaushal) [inline src/components/Sidebar/sidebarLists.js:67]: Please add the tags "ASCII" and "HEXADECIMAL" here and remove syntax and json from the list. The rest looks fine.

**Review by vaibhav-kaushal — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/sidebarLists.js (hunk 1 above)
Comment (vishalbhowmick) [reply src/components/Sidebar/sidebarLists.js]: @vaibhav-kaushal test and merge fast,  I am maintaining and updating this branch everyday with the main daily for 2 weeks. Can't do it for really long.

**Pull request merged**